<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemDialog</class>
 <widget class="QDialog" name="ItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>162</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>162</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>162</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MemeCollector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Meme type name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameInput"/>
   </item>
   <item>
    <widget class="QLabel" name="pathLabel">
     <property name="text">
      <string>Path</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pathInput"/>
   </item>
   <item>
    <widget class="QPushButton" name="openButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>nameInput</tabstop>
  <tabstop>pathInput</tabstop>
  <tabstop>openButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
